### PR TITLE
[doc] PR for Min CPU / mem requirements for Platform k8s (Doc-PLAT-2163)

### PR DESCRIPTION
**Pull request by lizayugabyte**
PR for https://yugabyte.atlassian.net/browse/PLAT-2163

Preview: https://deploy-preview-10484--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/install-yugabyte-platform/prerequisites/#hardware-requirements
and
https://deploy-preview-10484--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes/

### CODE REVIEW

**Comment by tylarb** [comment]
@lizayugabyte  content looks fine - presentation of the "minimum" vs "production" is not the same as the documentation on core db here: https://docs.yugabyte.com/latest/deploy/checklist/#hardware-requirements

I don't know if it's imperative that they look the same. I'll approve at this point, but the structure change is worth a short consideration.

**Review by tylarb — APPROVED** [review]

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md [hunk 1]
```diff
@@ -81,6 +81,13 @@ $ helm version
 
 The output should be similar to the following:
 
-```
+```output
 version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
 ```
+
+You can use Helm to configure the CPU and memory allocation for each Yugabyte Platform container. For example, the following command sets the container to 3 GB and Prometheus CPU to 3 cores:
```
Comment (iSignal) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md:88]: this should ideally go under https://docs.yugabyte.com/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes/#customization

**Review by iSignal — CHANGES_REQUESTED** [review]

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md:88]: Thanks for the suggestion. I checked the page, and this customization is already described there.
I deleted the content from install-yugabyte-platform/prepare-environment/kubernetes.md and cleaned up install-yugabyte-platform/install-software/kubernetes/#customization.

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md [hunk 2]
```diff
@@ -108,34 +108,33 @@ You install Yugabyte Platform on a Kubernetes cluster as follows:
...
-      --set yugaware.resources.limits.memory=4Gi
-    ```
+- You can change CPU and memory resources by executing a command similar to the following:
 
-1. To disable the internet/public facing LB, provide the annotations to YW service for disabling the public-facing LB.
+  ```sh
+  helm install yw-test yugabytedb/yugaware -n yb-platform \
+    --set yugaware.resources.requests.cpu=2 \
+    --set yugaware.resources.requests.memory=4Gi \
+    --set yugaware.resources.limits.cpu=2 \
```
Comment (iSignal) [inline docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md:121]: Let's include an example of customizing prometheus memory by adding something like
`prometheus.resources.requests.mem=6Gi`

**Review by iSignal — APPROVED** [review]
Looks good, one minor comment

File: docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md:121]: @iSignal I added a second example, but the command syntax is slightly different, so I'm not sure it's correct. Could you, please, check and let me know:

_You can change CPU and memory resources by executing a command similar to one of the following:_

```
helm install yw-test yugabytedb/yugaware -n yb-platform \
  --set yugaware.resources.requests.cpu=2 \
  --set yugaware.resources.requests.memory=4Gi \
  --set yugaware.resources.limits.cpu=2 \
  --set yugaware.resources.limits.memory=4Gi
```

```
**helm install --namespace yw-test yw-test yugabytedb/yugaware 
  --set yugaware.resources.requests.memory=3Gi, prometheus.resources.requests.cpu=3**
```

Comment (iSignal) [reply docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md:121]: You can collapse them into one example like below. 

``` 
helm install yw-test yugabytedb/yugaware -n yb-platform \
  --set yugaware.resources.requests.cpu=2 \
  --set yugaware.resources.requests.memory=4Gi \
  --set yugaware.resources.limits.cpu=2 \
  --set yugaware.resources.limits.memory=4G \
  --set prometheus.resources.requests.mem=6Gi
```

**Pull request merged**